schema/job.ts

```diff
@@ -0,0 +1,31 @@
+import z from "zod";
+
+export const saveJobsSchema = z.object({
+  companyName: z
+    .string()
+    .min(1, "Company name should contain atleast 1 character")
+    .max(50, "Company name should contain atmost 50 characters"),
+  jobTitle: z
+    .string()
+    .min(3, "Job title should contain atleast 3 character")
+    .max(50, "Job title should contain atmost 50 characters"),
+  jobDescription: z
+    .string()
+    .max(200, "Job Description should contain atmost 200 characters")
+    .optional(),
+  jobLocation: z
+    .string()
+    .min(3, "Location should contain atleast 3 characters")
+    .max(40, "Max location length is 40 characters."),
+  applicationUrl: z
+    .string()
+    .url("Provide a valid url")
+    .optional()
+    .or(z.literal("")),
+  remote: z.boolean().optional().default(false),
+  relocation: z.boolean().optional().default(false),
+  visa: z.boolean().optional().default(false),
+  jobType: z.enum(["full-time", "part-time", "freelancer", "other"]),
+});
```

Comment:
Would love some thoughts on if you think all of these are sensible too? 🦾

Reply:
I think yes, these should also be populated. As a result these values can be shown in the `jobs` page with their associated jobs.
A sample shown below present on [YC jobs page](https://www.workatastartup.com/companies?demographic=any&hasEquity=any&hasSalary=any&industry=any&interviewProcess=any&jobType=fulltime&layout=list-compact&role=eng&sortBy=created_desc&tab=any&usVisaNotRequired=any)

![image](https://github.com/user-attachments/assets/86de8ea6-f8fb-494c-b729-dbebb2284601)